libs/crosstarget-utils/src/common/spawn.rs

```diff
@@ -0,0 +1,12 @@
+use std::fmt::Display;
+
+#[derive(Debug)]
+pub struct SpawnError;
+
+impl Display for SpawnError {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "Failed to spawn a future")
+    }
+}
```

Comment:
and one more thing, really last this time, let's do this while we're here:

```suggestion
use derive_more::Display;

#[derive(Debug, Display)]
#[display(fmt = "Failed to spawn a future")]
pub struct SpawnError;
```